src/main/java/org/aesh/console/command/Command.java

```diff
@@ -19,13 +19,14 @@
  */
 package org.aesh.console.command;
```

Comment:
i think it would be good to move all these classes into org.aesh.command instead of org.aesh.console.command. what do you think?